src/docs/stan-reference/language.tex

```diff
@@ -1807,6 +1809,123 @@ \subsection{Operator Precedence and Associativity}
 is interpreted as \mbox{\code{-(n \textasciicircum\ 3)}}.
 
 
+\section{Conditional Operator}\label{conditional-operator.section}
+
+\subsection{Conditional Operator Syntax}
+
+The ternary conditional operator is unique in that it takes three
+arguments and uses a mixed syntax.  If \code{a} is an expression of
+type \code{int} and \code{b} and \code{c} are expressions that can be
```

Comment:
It'll match the rest of our conditionals and C++ better to
allow real there.  I don't mind either way.

- Bob

> On Jun 13, 2016, at 10:57 PM, Mitzi Morris <notifications@github.com> wrote:
> 
> In src/docs/stan-reference/language.tex:
> 
> > @@ -1807,6 +1809,123 @@ \subsection{Operator Precedence and Associativity}
> >  is interpreted as \mbox{\code{-(n \textasciicircum\ 3)}}.
> >  
> >  
> > +\section{Conditional Operator}\label{conditional-operator.section}
> > +
> > +\subsection{Conditional Operator Syntax}
> > +
> > +The ternary conditional operator is unique in that it takes three
> > +arguments and uses a mixed syntax.  If \code{a} is an expression of
> > +type \code{int} and \code{b} and \code{c} are expressions that can be
> 
> Daniel is correct - the code as written allows \code{a} (the condition) to be either int or real.
> do we want to restrict it to int?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>